cpp/src/strings/search/findall.cu

```diff
@@ -109,11 +109,11 @@ struct findall_count_fn : public findall_fn<stack_size> {
 }  // namespace
 
 //
-std::unique_ptr<table> findall_re(
+std::unique_ptr<table> findall(
   strings_column_view const& strings,
   std::string const& pattern,
-  rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource(),
-  rmm::cuda_stream_view stream        = rmm::cuda_stream_default)
+  rmm::cuda_stream_view stream,
```

Comment:
This should not have been defaulted. It could allow the caller to wrong stream accidentally. I wasn't specifically channeling 9854. This code is almost 3 years old now. We were defaulting the streams back then. It used to be the last parameter ... well there is a whole story that is not important here.